src/components/Button/InternalButton.tsx

```diff
@@ -0,0 +1,60 @@
+import { ElementType } from "react";
+import { classNames, variantClassNames } from "../../util/classes";
+import { ButtonProps } from "./Button";
+
+import { Button } from "./Button";
+
+const DEFAULT_BUTTON_TAG = "button" as const;
+
+export type InternalButtonProps<
+  T extends ElementType = typeof DEFAULT_BUTTON_TAG,
+> = Omit<ButtonProps<T>, "variant" | "size"> & InternalButtonOwnProps;
+
+export interface InternalButtonOwnProps {
+  variant: "wallet" | "wallet-inside";
+}
+
+export const InternalButton = <
+  T extends ElementType = typeof DEFAULT_BUTTON_TAG,
+>({
+  children,
+  className,
+  variant,
+  iconLeft,
+  ...props
+}: InternalButtonProps<T>) => {
+  return (
+    <Button
+      className={classNames(
+        variantClassNames(variant, {
+          wallet:
+            "ink-bg-background-light-transparent ink-px-1.5 ink-py-2 ink-text-body-2 ink-font-bold ink-text-default hover:ink-bg-background-light disabled:ink-bg-background-light-transparent-disabled disabled:ink-text-muted active:ink-bg-background-light",
+          "wallet-inside":
+            "ink-bg-background-light-invisible ink-px-1.5 ink-rounded-8 ink-text-body-2 ink-font-bold ink-text-default hover:ink-bg-background-container disabled:ink-bg-background-light-transparent-disabled disabled:ink-text-muted active:ink-bg-background-light",
+        }),
+        className
+      )}
+      size="sm"
+      {...(props as ButtonProps<T>)}
+    >
+      <div
+        className={classNames(
+          "ink-w-full ink-flex ink-items-center ink-gap-1.5",
+          variantClassNames(variant, {
+            wallet: "ink-justify-center",
+            "wallet-inside": "ink-justify-start",
+          })
+        )}
+      >
+        {iconLeft && (
+          <div className="ink-size-4 -ink-my-[10px] ink-flex ink-items-center ink-justify-center">
```

Comment:
`-ink-my-[10px]` is the only way I could get a 32px image down to 12px, matching the designs